### PR TITLE
add `isPolyfilled` properties to be able to interfaces (fixes #41)

### LINKED ISSUE #41: Consider setting `isPolyfilled` property on polyfilled `navigator.getVRDisplays`
It'd be nice if `navigator.getVRDisplays`/`navigator.getVRDevices` had a `isPolyfilled` boolean property.

### PR DESCRIPTION
**Pull request by cvan**

### CODE REVIEW

**Comment by cvan** [comment]
Per mrdoob/three.js#8330

**Comment by borismus** [comment]
Is legit to drop the .js prefix?

**Comment by cvan** [comment]
> Is legit to drop the .js prefix?

yep, with both ES6 and CommonJS modules

**Comment by borismus** [comment]
OK, good to know. However, I'd opt for consistency throughout the codebase. Let's stick to .js$ for now. Can change style in one fell swoop later.

**Comment by cvan** [comment]
> OK, good to know. However, I'd opt for consistency throughout the codebase. Let's stick to .js$ for now. Can change style in one fell swoop later.

yeah, that's smart. just fixed.

File: src/cardboard-vr-display.js [hunk 1]
```diff
@@ -31,6 +31,7 @@ var Eye = {
  * VRDisplay based on mobile device parameters and DeviceMotion APIs.
  */
 function CardboardVRDisplay() {
+  this.isPolyfilled = true;
```
Comment (cvan) [inline src/cardboard-vr-display.js]: I just realised I can remove these since these displays inherit from `VRDisplay`, which also sets the prop

Comment (borismus) [inline src/cardboard-vr-display.js]: Sounds good, you'll update this PR?

Comment (borismus) [inline src/cardboard-vr-display.js]: Ping!

**Comment by cvan** [comment]
updated!

**Comment by borismus** [comment]
Thanks

**Pull request merged**